src/main/scala/Week1/HelloWorld.sc

```diff
@@ -0,0 +1,205 @@
+import jdk.javadoc.internal.doclets.toolkit.util.DocFinder.Input
+//task one - HELLO WORLD!
+println("hello world")
+
+//task two - ASSIGNMENT
+var output:String = "Hello World"
+println(output)
+
+//task three - REASSIGNMENT
+var outputA:String = "outputA value 1"
+val outputB:String = "outputB value 1"
+
+outputA = "outputA value 2"
+//outputB = "outputB value 2"
+// can't change because val is immutable
+println(outputA)
+println(outputB)
+
+
+//task 4 - PARAMETERS
+showInput("hello world")
+def showInput(input:String) = {println(input)}
+
+//task 5 - PARAMETERS 2
+showInputWithDefault("");
+
+def showInputWithDefault(input:String) = {
+  if (input != null && input != "" ){
+  println(input)
+  }
+  else{
+    println("default")
+  }
+}
+
+//task 6 - PARAMETERS 3
+var timesToRepeat:Int = 3
+var inputToRepeat:String = "hello"
+
+stringMult(inputToRepeat, timesToRepeat)
+
+def stringMult(input:String, mult :Int) ={
+  for( a <- 0 to mult){
+    println(input)
+  }
+}
+
+//task 7 - Return Types
+println(returnHelloWorld())
+def returnHelloWorld():String = {"Hello World"}
+
+//task 8 - TYPE INFERENCE
+returnAll(1)
+returnAll("one")
+returnAll(true)
+returnAll(1.1)
+def returnAll(input:Any) = {println(input)}
+
+//task 9 -STRINGS
+var inputToTrim = "hello"
```

Comment:
inputToTrim and trimBy should be val, as you're not reassigning the value to them, therefore no point in having them as var's